packages/familie-form-elements/form-elements.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { FamilieCheckbox } from './src';
+
+export default {
+    component: FamilieCheckbox,
+    parameters: {
+        componentSubtitle: 'Visittkort brukes til å vise enkel informasjon om en bruker.',
+    },
+    title: 'Komponenter/Visittkort',
+};
+
+export const familieCheckbox = () => {
```

Comment:
Hvis du klarer å lage en komponent som rendrer alle form-elements(på sikt) så kunne vi hatt en toggle for å toggle lesevisning 🙌